### PR TITLE
Update/Extend git rebase

### PR DESCRIPTION
**Pull request by deathaxe**
Hi, made up my mind about git rebase syntax a bit. Here's the result in case you are interested.

### CODE REVIEW

**Comment by deathaxe** [comment]
Ready to merge.

File: Git/gitrebase.sublime-syntax [hunk 1]
```diff
@@ -2,57 +2,136 @@
...
+      scope: invalid.illegal.hash-expected.git.rebase
+      pop: true
+    - match: $
+      pop: true
+
+  commit-message:
+    - meta_scope: meta.commit.git.rebase
+    - match: \n
+      pop: true
+    - match: (?=\S)  # subject beginns with first none whitespece
```
Comment (y0ssar1an) [inline Git/gitrebase.sublime-syntax]: typo: beginns

File: Git/gitrebase.sublime-syntax [hunk 2]
```diff
@@ -2,57 +2,136 @@
...
+        - include: commands-msg
         - include: rebase-msg
 
-  # e.g. pick d284bb2 Initial commit
-  commit:
-    - match: ^\s*(squash|reword|pick|fixup|exec|edit|drop|x|s|r|p|f|e|d)[\t ]
+  head-line:
+    # all comments ending with colon are scoped as headlines
+    - match: \b\w.+(?=:\s*$)
+      scope: markup.heading.git.rebase
```
Comment (y0ssar1an) [inline Git/gitrebase.sublime-syntax]: nit: You don't have to hyphenate `head-line`.

**Comment by y0ssar1an** [comment]
beautiful work 👍

**Pull request merged**